src/Avalonia.DotNetCoreRuntime/Avalonia.DotNetCoreRuntime.csproj

```diff
@@ -8,6 +8,7 @@
     <ProjectReference Include="..\Avalonia.Controls\Avalonia.Controls.csproj" />
     <ProjectReference Include="..\Avalonia.Visuals\Avalonia.Visuals.csproj" />
     <ProjectReference Include="..\Gtk\Avalonia.Gtk3\Avalonia.Gtk3.csproj" />
+    <ProjectReference Include="..\Avalonia.Windowing\Avalonia.Windowing.csproj" />
```

Comment:
This project doesn't exist in this PR. Can we revert the change to this file?

Reply:
fixed